src/plugins/event-plugins/WebVitalsPlugin.ts

```diff
@@ -34,28 +47,66 @@ export class WebVitalsPlugin extends InternalPlugin {
     configure(config: any): void {}
 
     protected onload(): void {
+        this.context.eventBus.subscribe(Topic.EVENT, this.lcpHelper); // eslint-disable-line
         onLCP((metric) => this.handleLCP(metric));
         onFID((metric) => this.handleFID(metric));
         onCLS((metric) => this.handleCLS(metric));
     }
 
-    handleLCP(metric: LCPMetricWithAttribution | Metric) {
+    private lcpHelper(msg: any) {
+        const event = msg as ParsedRumEvent;
```

Comment:
```suggestion
    private lcpHelper(event: ParsedRumEvent) {
```

Reply:
I remember typescript was complaining when I tried this initially. Double checking to see if this suggestion works